cmd/trade.go

```diff
@@ -70,6 +70,25 @@ func init() {
 		}
 		log.Printf("Trading %s:%s for %s:%s\n", botConfig.ASSET_CODE_A, botConfig.ISSUER_A, botConfig.ASSET_CODE_B, botConfig.ISSUER_B)
 
+		//Add current strategy to the log
+		var strat_display string
+		switch *strategy {
```

Comment:
We can do without the switch-case to keep it simple. Just use *strategy instead directly in the log statement.